UI/ViewModels/MainWindowViewModel.cs

```diff
@@ -198,6 +223,12 @@ public MainWindowViewModel()
             .Filter(x => string.IsNullOrWhiteSpace(SearchText) || x.Name.ToLower().Contains(SearchText.ToLower()))
             .Bind(out _requestTemplatesFiltered)
             .Subscribe();
+        _mocks.Connect()
+            .AutoRefreshOnObservable(_ => this.ObservableForProperty(x => x.SearchText))
+            .Sort(SortExpressionComparer<MockTemplate>.Ascending(t => t.Name))
+            .Filter(x => string.IsNullOrWhiteSpace(SearchText) || x.Name.ToLower().Contains(SearchText.ToLower()))
```

Comment:
Filter first, Sort after, otherwise we're filtering sorted results, but sorting filtered results makes more sense